src/server/serving.ts

```diff
@@ -169,6 +173,14 @@ export class Server {
       res.status(200).send('ok');
     });
 
+    // get Twitter API Version
+    this.app.get('/get_twitter_api_version', (_req, res) => {
+      res.status(200).send({
+        version:this.config.twitterApiCredentials.useEssentialOrElevatedV2 ? TwitterApiVersion.essentialOrElevatedV2 : TwitterApiVersion.enterprise
+      });
+
```

Comment:
Make the code readable

```suggestion
      const version = this.config.twitterApiCredentials.useEssentialOrElevatedV2 ? TwitterApiVersion.essentialOrElevatedV2 : TwitterApiVersion.enterprise;
      res.status(200).send({version});
```